### PR TITLE
Add quickstart

### PR DESCRIPTION
**Pull request by lewtun**
# What does this PR do?

This PR adds a quickstart for the ONNX Runtime features of the library. Although the docstrings are great, I think this will help people get started quickly :)

I also took the liberty of tidying up the installation section & included the `graphcore` install.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
...
+python -m pip install optimum
+```
 
-🤗 Optimum with [Intel Neural Compressor (INC)](https://github.com/intel/neural-compressor) or [ONNX runtime](https://github.com/microsoft/onnxruntime) dependencies can be installed respectively using pip as follows:
+If you'd like to use the accelerator-specific features of 🤗 Optimum, you can install the required dependencies according to the table below:
 
-`pip install optimum[intel]`
+| Accelerator                                                                 | Installation                                 |
+|:----------------------------------------------------------------------------|:---------------------------------------------|
+| [ONNX runtime](https://github.com/microsoft/onnxruntime)                    | `python -m pip install optimum[onnxruntime]` |
```
Comment (lewtun) [inline README.md:48]: I decided to create a table here to accommodate all the upcoming integrations :)

Comment (echarlaix) [reply README.md:48]: Great idea, I love it !

File: README.md [hunk 2]
```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
...
+ort_outputs.predictions
+```
+
+Similarly, you can apply static quantization by simply changing the `quantization_approach` in the `ORTConfig` object:
+
+```python
+ort_config = ORTConfig(quantization_approach="dynamic")
+```
+
+Static quantization relies on feeding batches of data through the model to identify which activations should be quantized. To support this, 🤗 Optimum allows you to provide a _calibration dataset_. The calibration dataset can be a simple `Dataset` object from the 🤗 Datasets library, or any dataset that's hosted on the Hugging Face Hub. For this example, we'll pick the [`sst2`](https://huggingface.co/datasets/glue/viewer/sst2/test) dataset that the model was originally trained on:
```
Comment (echarlaix) [inline README.md:113]: I think "to identify which activations should be quantized" should be replaced by something like "to estimate the activations quantization parameters"

File: README.md [hunk 3]
```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
...
+dataloader = DataLoader(tokenized_ds)
+ort_outputs = ort_model.evaluation_loop(dataloader)
+# Extract logits!
+ort_outputs.predictions
+```
+
+Similarly, you can apply static quantization by simply changing the `quantization_approach` in the `ORTConfig` object:
+
+```python
+ort_config = ORTConfig(quantization_approach="dynamic")
```
[the remark this answers is not in the document]
Comment (lewtun) [reply README.md:110]: Good catch!

File: README.md (hunk 1 above)
Comment (echarlaix) [reply README.md:48]: Great idea, I love it !

File: README.md [hunk 4]
```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
...
+| [ONNX runtime](https://github.com/microsoft/onnxruntime)                    | `python -m pip install optimum[onnxruntime]` |
+| [Intel Neural Compressor (INC)](https://github.com/intel/neural-compressor) | `python -m pip install optimum[intel]`       |
+| [Graphcore IPU](https://www.graphcore.ai/products/ipu)                      | `python -m pip install optimum[graphcore]`   |
 
-`pip install optimum[onnxruntime]`
 
-If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you must install the library from source:
+If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you can install the library from source as follows:
+
+```bash
```
[the remark this answers is not in the document]
Comment (lewtun) [reply README.md:55]: I've now added a piece about appending the `#egg` piece to the source install

**Comment by lewtun** [comment]
Thanks for the review @michaelbenayoun @echarlaix ❤️ ! I've integrated your feedback so will merge once the tests pass

**Pull request merged**